bin/wc_win.mak

```diff
@@ -6,46 +6,12 @@
 #  Usage: 'wmake -h -f wc_win.mak'
 #
 
-.EXTENSIONS:
-.EXTENSIONS: .exe .obj .c
+MODEL   = win32
+CC      = *wcc386 -3r
+CFLAGS  = -bt=nt -mf -oaxt -DWIN32 -DWATT32_STATIC
```

Comment:
Why are Win32 sample not using `watt-32.dll`?

Reply:
It was first atempt to do working makefiles.
As I see you extended wc_win.mak to switch to use static or dynamic library version for compilation.

Reply:
Some issue with Watcom `tracert.exe` when linked to `watt-32.dll`. `_arp_resolve` not found !?